.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -578,14 +580,14 @@ stages:
       displayName: Download MSI
       inputs:
         artifact: windows-msi-$(targetPlatform)
-        patterns: '**/*.msi'
+        patterns: '**/*profiler-beta.msi'
         path: $(System.DefaultWorkingDirectory)/$(relativeMsiOutputDirectory)
 
     - script: tracer\build.cmd BuildWindowsIntegrationTests -Framework $(framework)
       displayName: BuildWindowsIntegrationTests
 
     - script: |
-        docker-compose build --build-arg dotnet_tracer_msi=.$(relativeMsiOutputDirectory)/*.msi --build-arg ENABLE_32_BIT=$(enable32bit) IntegrationTests.IIS
+        docker-compose build --build-arg dotnet_tracer_msi=.$(relativeMsiOutputDirectory)/*profiler-beta.msi --build-arg ENABLE_32_BIT=$(enable32bit) IntegrationTests.IIS
```

Comment:
Not needed for this particular PR. But we'll need to drop this MSI name in favor of the other one before releasing. 
Also, this raises one point I had forgotten, I assume there's no issue to build the msi with profiler for x86 platforms even though you don't support it, right?

Reply:
> Not needed for this particular PR. But we'll need to drop this MSI name in favor of the other one before releasing.

👍 

> Also, this raises one point I had forgotten, I assume there's no issue to build the msi with profiler for x86 platforms even though you don't support it, right?

Event though we do not support it, the profiler can be build also for x86 platforms. If anything fails in the CI for x86, we fix it as if it was x64.